pkgs/development/python-modules/dnspython/default.nix

```diff
@@ -24,10 +26,14 @@ buildPythonPackage rec {
   disabledTests = [
     # dns.exception.SyntaxError: protocol not found
     "test_misc_good_WKS_text"
+    # fails if IPv6 isn't available
+    "test_resolver_override"
```

Comment:
Do we need to remove this test? If I recall correctly this builds on hydra

Reply:
You are correct that it does build on Hydra, but it won't build locally on my machine, since the network I'm on doesn't provide IPv6. So seems good to disable it in case people end up building locally.